<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogConvert</class>
 <widget class="QDialog" name="DialogConvert">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>267</width>
    <height>286</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="maximumSize">
   <size>
    <width>366</width>
    <height>286</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Конвертер величин</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../../resourse.qrc">
    <normaloff>:/resourse/default/converter.png</normaloff>:/resourse/default/converter.png</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="2" column="1">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>70</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="11" column="0">
    <widget class="QLabel" name="label_7">
     <property name="text">
      <string>Длинна волны (м)</string>
     </property>
    </widget>
   </item>
   <item row="7" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>дБм</string>
     </property>
    </widget>
   </item>
   <item row="2" column="2">
    <widget class="QLineEdit" name="lineEdit_outDBm">
     <property name="frame">
      <bool>false</bool>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="7" column="2">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Ватт</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="DotLineEdit" name="lineEdit_inWatt">
     <property name="cursor">
      <cursorShape>IBeamCursor</cursorShape>
     </property>
     <property name="inputMethodHints">
      <set>Qt::ImhNone</set>
     </property>
     <property name="inputMask">
      <string/>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>дБм</string>
     </property>
    </widget>
   </item>
   <item row="8" column="0">
    <widget class="QLineEdit" name="lineEdit_inDBm"/>
   </item>
   <item row="10" column="0">
    <widget class="QLineEdit" name="lineEdit_inFreq"/>
   </item>
   <item row="10" column="2">
    <widget class="QLineEdit" name="lineEdit_outDlvoln">
     <property name="frame">
      <bool>false</bool>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="9" column="2">
    <widget class="QLabel" name="label_6">
     <property name="text">
      <string>Длинна волны (м)</string>
     </property>
    </widget>
   </item>
   <item row="0" column="2" alignment="Qt::AlignLeft">
    <widget class="QLabel" name="label_10">
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Результат:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="11" column="2">
    <widget class="QLabel" name="label_8">
     <property name="text">
      <string>Частота (МГц)</string>
     </property>
    </widget>
   </item>
   <item row="13" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="12" column="0">
    <widget class="QLineEdit" name="lineEdit_inDlvoln"/>
   </item>
   <item row="8" column="2">
    <widget class="QLineEdit" name="lineEdit_outWatt">
     <property name="frame">
      <bool>false</bool>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="12" column="2">
    <widget class="QLineEdit" name="lineEdit_outFreq">
     <property name="frame">
      <bool>false</bool>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Ватт</string>
     </property>
    </widget>
   </item>
   <item row="9" column="0">
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>Частота (МГц)</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0" alignment="Qt::AlignLeft">
    <widget class="QLabel" name="label_9">
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Исходное значение:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="14" column="2">
    <widget class="QPushButton" name="pushButton">
     <property name="text">
      <string>Закрыть</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>DotLineEdit</class>
   <extends>QLineEdit</extends>
   <header location="global">../Widget/DotLineEdit.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../resourse.qrc"/>
  <include location="../../resourse.qrc"/>
 </resources>
 <connections/>
</ui>
